ql/src/test/results/clientpositive/llap/authorization_insert_column_positive.q

```diff
@@ -0,0 +1,69 @@
+set hive.security.authorization.manager=org.apache.hadoop.hive.ql.security.authorization.plugin.sqlstd.SQLStdHiveAuthorizerFactoryForTest;
```

Comment:
I think this file should be in `queries/clientpositive/`, not `results/clientpositive/llap`

Reply:
Move it to queries/clientpositive/